chat_shell/chat_shell/tools/builtin/knowledge_base.py

```
@@ -106,6 +122,292 @@ def injection_strategy(self) -> InjectionStrategy:
+                        configs[kb_id] = {
+                            "maxCallsPerConversation": spec.get(
+                                "maxCallsPerConversation",
+                                DEFAULT_MAX_CALLS_PER_CONVERSATION,
+                            ),
+                            "exemptCallsBeforeCheck": spec.get(
+                                "exemptCallsBeforeCheck",
+                                DEFAULT_EXEMPT_CALLS_BEFORE_CHECK,
+                            ),
+                            "name": spec.get("name", f"KB-{kb_id}"),
+                        }
+                except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)